dependencies/keycloak/deployment/kustomization.yml

```diff
@@ -23,3 +23,16 @@ patches:
     target:
       kind: Deployment
       name: keycloak-operator
+  - patch: |
+      - op: add
+        path: /spec/template/spec/containers/0/resources
+        value:
+          requests:
+            cpu: 50m
+            memory: 50Mi
+          limits:
+            cpu: 100m
+            memory: 100Mi
```

Comment:
@gbenhaim do you expect this to be enough?

Reply:
i see now that the sanity test failed with the following error:
```kubectl wait --for=condition=Ready --timeout=300s -n keycloak -l app.kubernetes.io/name=keycloak-operator pod: Keycloack did not become available within the allocated time.```

I'll increase the amount to see if it solves the issue (my guess is that keycloak didn't have enough memory or cpu to become available in time)

Reply:
Yes. Keycloak is quite heavy.

Reply:
I raised the quota to:
``` requests:
            cpu: 200m
            memory: 200Mi
          limits:
            cpu: 300m
            memory: 300Mi```
           and now it works, the sanity test succeeded

Reply:
and now it works, the sanity test succeeded